t/Util.pm

```diff
@@ -226,6 +226,12 @@ sub spawn_h2o {
         $max_ssl_version = $conf->{max_ssl_version} || undef;
         $conf = $conf->{conf};
     }
+    my $user;
+    if ($conf =~ /^user:/) { # do not override the specified user
+        $user = "";
```

Comment:
Thank you for the PR. Makes sense, I only have one question.

Would it be possible accept user as part of `$conf` like `$conf->{max_ssl_version}`? I ask this because that's the convention that we've adopted so far. We haven't really parsed the given config string to see if something has to be changed.

WDYT?

Reply:
That's a good question! 

I believe it's better than this potentially unreliable check with `/^user:/`.  This change is just because I hesitated to rewrite the tests that already used `user:`.

Fixed at 11612477f69f3ebf06afaf9110ee97358620f77d